config/environments/production.rb

```diff
@@ -57,7 +57,24 @@
   config.log_tags = [:request_id]
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  if ENV['REDIS_URL'] && ENV['REDIS_PORT']
+    config.cache_store = :redis_cache_store, {
+      url: ENV['REDIS_URL'], port: ENV['REDIS_PORT'], db: 0, namespace: 'cache',
+      expires_in: 90.minutes
+    }
+
+    config.session_store :redis_store,
+                         servers: [{
+                           url: ENV['REDIS_URL'],
+                           port: ENV['REDIS_PORT'],
+                           db: 0,
+                           namespace: 'session'
+                         }],
+                         expire_after: 90.minutes,
+                         key: "_#{Rails.application.class.module_parent_name.downcase}_session",
+                         threadsafe: true,
+                         secure: true
+  end
```

Comment:
could we move all this into config/applicaiton.rb so we don't have duplication between development and production? I also think you've committed a lot more to development.rb than you meant to?

Reply:
Sure thing and nope, commited some bits that help along with development and have little impact on day-to-day of the system